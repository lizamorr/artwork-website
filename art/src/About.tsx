import React, {Component} from "react";

class About extends Component {
    render() {
        return (
            <></>
        );
    }
}

export default About;