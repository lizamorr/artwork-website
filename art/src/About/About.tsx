import React, { Component } from 'react';

class About extends Component {
  render() {
    return <>Hello!</>;
  }
}

export default About;
