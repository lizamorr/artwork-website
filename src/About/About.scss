.about-container {
  margin: 65px 5% 0% 5%;
  display: flex;
  flex-wrap: wrap;
  justify-content: center;
  .img-follow {
    display: flex;
    flex-direction: column;
    margin-left: 20px;
    margin-bottom: 20px;
    margin-right: 20px;
  }
  .about-paragraph {
    flex: 2;
    display: flex;
    justify-content: center;
    img {
      max-width: 450px;
      min-width: 237px;
      height: auto;
      width: 100%;
    }
  }
  .paris-photo {
    flex: 1;
    border-radius: 10px;
    display: block;
    max-width: 350px;
    max-height: 500px;
    width: 100%;
    height: auto;
  }
  .instagram-container {
    display: flex;
    justify-content: center;
    margin-top: 10px;
    .instagram {
      width: 25px;
      height: 25px;
      object-fit: cover;
    }
  }
}
