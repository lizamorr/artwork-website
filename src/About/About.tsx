import React from 'react';
import parisPhoto from './me-paris.jpg';

const About: React.FC = () => {
  return (
    <div className="about-container">
      <img src={parisPhoto} alt="Me in Paris" />
      <div className="about-text">
        <div className="about-title">Liza Morrison</div>
        <div className="about-subtitle">Artist & Software Engineer</div>
        <div className="about-paragraph">
          Born and raised in New Jersey, I swapped pre-school for art classes
          (thanks mom) and continued scribbling on paper. I received my BS in
          Chemistry from the University of Richmond and claimed Richmond as my
          second home. After my undergraduate studies, I worked as an associate
          analytical scientist, web developer, and software engineer. When not
          acting as Mr. Robot, I'm an artist. The majority of my work after high
          school has been commissions (thanks to all my awesome customers!). My
          specialities range across various mediums: graphite, charcoal, pen,
          oils, acrylic, watercolor, and colored pencils. I currently reside in
          Belgium with my partner and our three pets (2 dogs and 1 cat). Please
          contact me with more information about commissions. Thank you for
          checking out my work.
        </div>
        <div className="about-follow">
          Follow:{' '}
          <a href="https://www.instagram.com/lizamorrisonart/">
            @Lizamorrisonart
          </a>
        </div>
      </div>
    </div>
  );
};

export default About;
