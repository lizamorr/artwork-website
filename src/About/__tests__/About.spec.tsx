import About from '../About';
import renderer from 'react-test-renderer';
import React from 'react';

describe('About Component', () => {
  it('should render correctly', () => {
    const component = renderer.create(<About />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
