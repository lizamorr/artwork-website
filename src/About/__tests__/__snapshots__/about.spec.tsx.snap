// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Component should render correctly 1`] = `
Array [
  <div
    style={
      Object {
        "display": "block",
      }
    }
  >
    <div
      className="spinner"
    >
      <div
        className="spinner-border"
        role="status"
      >
        <span
          className="sr-only"
        >
          Loading...
        </span>
      </div>
    </div>
  </div>,
  <div
    style={
      Object {
        "display": "none",
      }
    }
  >
    <div
      className="about-container"
    >
      <div
        className="img-follow"
      >
        <img
          alt="Me in Paris"
          className="paris-photo"
          onLoad={[Function]}
          src="test-file-stub"
        />
        <div
          className="instagram-container"
        >
          <a
            href="https://instagram.com/lizamorrisonart/"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="Instagram"
              className="instagram"
              src="https://static.wixstatic.com/media/01c3aff52f2a4dffa526d7a9843d46ea.png/v1/fill/w_50,h_50,al_c,q_85,usm_0.66_1.00_0.01/01c3aff52f2a4dffa526d7a9843d46ea.webp"
            />
          </a>
        </div>
      </div>
      <div
        className="about-paragraph"
      >
        <img
          src="test-file-stub"
        />
      </div>
    </div>
    <div
      className="footer"
      id="footer"
    >
      Â© Liza Morrison 2021
    </div>
  </div>,
]
`;
